src/modules/uuv_att_control/uuv_att_control.cpp

```diff
@@ -236,7 +236,15 @@ void UUVAttitudeControl::Run()
 			}
 
 			/* Geometric Control*/
-			control_attitude_geo(attitude, _attitude_setpoint, angular_velocity, _rates_setpoint);
+			int skip_controller = _param_skip_ctrl.get();
+
+			if (skip_controller) {
+				constrain_actuator_commands(_rates_setpoint.roll, _rates_setpoint.pitch, _rates_setpoint.yaw,
+							    _rates_setpoint.thrust_body[0]);
```

Comment:
This looks like it is using rates setpoints that are already supported through offboard mode - and not actuator controls. Am I missing something?

Reply:
I'm not sure whether we are talking past each other or if i misunderstand something.

Offboard enables me to send setpoints via mavlink. But the status quo is, that the `uuv_att_control` module calls its `control_attitude_geo()` method. 

My newly introduced parameter makes it now possible to bypass this call to `control_attitude_geo()`.

**Status quo**: offboard mode is true -> companion computer sends setpoints -> `uuv_att_control`-module gets these setpoints and calls its `control_attitude_geo()` method -> the controller computes some output that is published as `actuator_controls_0` message.

**New Solution**: offboard mode is true -> companion computer sends setpoints -> `uuv_att_control`-module gets these setpoints and if SKIP_CTRL is true bypasses the call to `control_attitude_geo()` -> directly outputs the value of the setpoint as `actuator_controls_0` message.

If this behaviour can already be achieved by existing modes/flags, can you please give me a hint how to accomplish it?